lib/bulk_data_test_kit/v2.0.0/client/docs/suite_description.md

```diff
@@ -0,0 +1,10 @@
+The Bulk Data Access v2.0.0 Client test suite validates the conformance of a client
+to the [FHIR Bulk Data Access IG STU2](http://hl7.org/fhir/uv/bulkdata/STU2).
+
```

Comment:
I think there should be a section in this description file that explains how to run these tests with the postman collection. See the CARIN IG for Blue Button client suite description as an example [here](https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/main/lib/carin_for_blue_button_test_kit/docs/carin_for_blue_button_v2.0.0_client_suite_description.md)

Reply:
Good idea; I've added this section